sklearn/ensemble/gradient_boosting.py

```diff
@@ -157,7 +193,26 @@ def predict(self, X):
 
 
 class LogOddsEstimator(object):
-    """An estimator predicting the log odds ratio."""
+    """An estimator predicting the log odds ratio.
+
+    This estimator takes only the binary target labels of the training set into
+    account. The prediction is independent of the X values in the training set
+    and those passed to the :func:`predict` function.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.ensemble import gradient_boosting
+    >>> y = np.array([1, 1, 0, 1])
+    >>> est = gradient_boosting.LogOddsEstimator()
+    >>> est.fit(np.random.rand(5, 2), y)
+    >>> est.prior
+    1.0986122886681098
```

Comment:
it seems sufficient to use it once per file but since I find it confusing to see the ellipsis without the doctest hint I kept the other hint in place (although redundant).